columnflow/tasks/framework/mixins.py

```diff
@@ -188,6 +188,11 @@ def calibrator_inst(self) -> Calibrator:
 
         return self._calibrator_inst
 
+    @property
+    def calibrator_repr(self):
+        """Return the name of the calibrator."""
```

Comment:
This should state

```
"""
Return a string representation of the calibrator.
"""
```

If it was always the name, there would be no need for a customization hook / property.
Same below.